### ISSUE TITLE
Updated README for Docker Build

### ISSUE THREAD

**Issue by Mentors4EDU** [bitclout/core#3]
Hi, please use the command `docker build -t ubuntu-test:latest` over the `docker build Dockerfile` command or update the contexts, please [see here](https://stackoverflow.com/questions/43296019/docker-build-gives-unable-to-prepare-context-context-must-be-a-directory-use)

**Comment by Mentors4EDU** [comment]
Yeah it looks like it needs to be tagged to run correctly or the `docker build Dockerfile` won't run well.

See screenshot below:

![Screenshot from 2021-05-18 13-52-52](https://user-images.githubusercontent.com/12502053/118700174-603b7a00-b7e0-11eb-97ec-ebca21f90be0.png)

**Comment by maebeam** [comment]
Where do you see a reference to `docker build Dockerfile`?

**Comment by Mentors4EDU** [comment]
Hi @maebeam I'm not quite sure. I believe I seen it somewhere but forgot. 
Either way, please add the `docker build -t ubuntu-test:latest` command to the README so that people don't use the other command by default.

**Comment by Galanoz** [comment]
unable to prepare context: context must be a directory: /Users/tempUser/git/docker/Dockerfile

**Comment by bloated-devish** [comment]
https://github.com/bitclout/mongodb-dumper suggests docker build to be run.
